.github/workflows/lint-test.yml

```diff
@@ -205,7 +222,7 @@ jobs:
         TZ: "America/New_York"
       run: |
         echo "$GITHUB_STEP_SUMMARY_HEADER" | sed "s/#name#/Test Summary/" >> $GITHUB_STEP_SUMMARY
-        nox -s test-${{ matrix.python-version }} -- --color=no --cov-report= | tee -a $GITHUB_STEP_SUMMARY
+        nox --force-python ${{ matrix.python-version }} -s test -- --color=no --cov-report= | tee -a $GITHUB_STEP_SUMMARY
```

Comment:
nox tries to invoke pytest three times, interpreting `3.11.0-alpha - 3.11` as three separate versions (`3.11.0-alpha`, `-`, `3.11`), which is why the check is currently failing.

n.b. this doesn't appear to be an issue for the pyright job, as pyright essentially only uses `version.split(".")[:2]`

Reply:
lmao will fix

Reply:
fixed in https://github.com/DisnakeDev/disnake/pull/785/commits/6479df85866993ffea5a864e11c6dbcd184bb6fe